### PR TITLE
add semver governance page

### LINKED ISSUE #376: Add supporting information for digest governance in 0.8
### Add label "documentation-id"
- Add the `documentation-id` bug label to this issue.

### What version of Kabanero does this apply to?
- Latest __
  or
- Specify versions: _____

### Specify what you want documented (required):
Complete the following information:

#### To create a topic:
- Specify a link to the section in the navigation where the topic should go:
- Specify a first draft of the topic that you want added:

#### To update existing topics:
- Specify links to the topics that are affected:
- Include a copy of the current text and the exact text to which it will change. For example: Change ABC to XYZ:

### PR DESCRIPTION
**Pull request by jantley-ibm**
closes #376

### CODE REVIEW

**Comment by jantley-ibm** [comment]
Thanks @SueChaplain -- these changes have been made.

File: ref/general/reference/semver-governance.adoc [hunk 1]
```diff
@@ -94,20 +94,24 @@ triggered to build or to deploy applications built on the new patch level.
...
-4. Rebuild the stack-hub index shared with developers to include the latest patch level.
-5. Build pipelines label application containers, which identify the three-digit semantic version application stack that is used to
-build the application. Query your container registry for containers that are built with the older patch level.
-6. For each container, rebuild, test, and redeploy the applications on the older patch level.
+1. Build the new application stack container. The build process tags the container with its desired 3-digit semantic version tag
+(e.g., `0.2.23`).
+2. Add the stack index.yaml to the Kabanero custom resource (CR) of the desired deployments to activate the new stack.
+3. Release the stack and push the images. As part of this process, the existing tags `latest`, MAJOR and MAJOR.MINOR are updated
+to the new patch container (i.e., `0.2.23`), although you can also make these changes manually.
+4. Update the Kabanero CR that is shared with developers with the new index that includes the latest patch level. Do not do this
```
Comment (stephenkinder) [inline ref/general/reference/semver-governance.adoc:102]: Update the Kabanero CR with the new index that is shared with indexes (if necessary) which includes this new patched level.

File: ref/general/reference/semver-governance.adoc [hunk 2]
```diff
@@ -94,20 +94,24 @@ triggered to build or to deploy applications built on the new patch level.
...
+step before the stack is activated, otherwise developers could create and push an application using the newer stack before it is
```
Comment (stephenkinder) [inline ref/general/reference/semver-governance.adoc:103]: So whether this is a "problem" is actually dependent on the selected governance policy.

File: ref/general/reference/semver-governance.adoc [hunk 3]
```diff
@@ -94,20 +94,24 @@ triggered to build or to deploy applications built on the new patch level.
...
+available.
+5. Build pipelines label the containers created for the microservice with the identity of the three-digit application stack
+that is used to build the application. Query your container registry for containers that are built with the older patch level.
+6. For each updated container, rebuild, test, and redeploy the applications on the new patch level.
```
Comment (stephenkinder) [inline ref/general/reference/semver-governance.adoc:107]: I thought we said 5 & 6 steps moved down to the retiring an older patch level.

File: ref/general/reference/semver-governance.adoc (hunk 1 above)
Comment (jantley-ibm) [reply ref/general/reference/semver-governance.adoc:102]: I think you meant "index that is shared with developers (if necessary)" -- if not, let me know

File: ref/general/reference/semver-governance.adoc (hunk 2 above)
Comment (jantley-ibm) [reply ref/general/reference/semver-governance.adoc:103]: will note

File: ref/general/reference/semver-governance.adoc (hunk 3 above)
Comment (jantley-ibm) [reply ref/general/reference/semver-governance.adoc:107]: Fixed. Step 6 and step 1 of the next section appear the same, consolidating.

**Pull request merged**